test/ejabberd_tests/tests/privacy_helper.erl

```diff
@@ -114,100 +121,129 @@ is_presence_error(Stanza) ->
     andalso
     escalus_pred:is_error(<<"modify">>, <<"not-acceptable">>, Stanza).
 
+privacy_list({Name, #client{} = Target}) ->
+    JID = escalus_utils:get_short_jid(Target),
+    escalus_stanza:privacy_list(Name, list_content(Name, JID));
 privacy_list(Name) ->
     escalus_stanza:privacy_list(Name, list_content(Name)).
 
+
 % Geralt is not used by the privacy tests, but lists have to have
 % at least one element so we use him for a no-op.
 list_content(<<"noop_list">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, geralt, [])
-    ];
-list_content(<<"deny_bob">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [])
-    ];
-list_content(<<"allow_bob">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"allow">>, bob, [])
-    ];
-list_content(<<"deny_bob_message">>) -> [
-        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>, bob, [<<"message">>])
+        escalus_stanza:privacy_list_jid_item(<<"1">>, <<"deny">>,
+            <<"geralt@localhost">>, [])
     ];
 list_content(<<"deny_group_message">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>, <<"ignored">>, [<<"message">>])
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"group">>,
+            <<"ignored">>, [<<"message">>])
     ];
 list_content(<<"deny_unsubscribed_message">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>, <<"none">>, [<<"message">>])
+        escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, <<"subscription">>,
+            <<"none">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message">>) -> [
         escalus_stanza:privacy_list_item(<<"1">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_from">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"from">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"from">>, [<<"message">>]),
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_to">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"to">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"to">>, [<<"message">>]),
         escalus_stanza:privacy_list_item(<<"2">>, <<"deny">>, [<<"message">>])
     ];
 list_content(<<"deny_all_message_but_subscription_both">>) -> [
-        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>, <<"both">>, [<<"message">>]),
+        escalus_stanza:privacy_list_item(<<"1">>, <<"allow">>, <<"subscription">>,
+            <<"both">>, [<<"message">>]),
```

Comment:
It's the last one re indentation but there are similar lines below. Hope you'll find them.